Source/JavaScriptCore/llint/LowLevelInterpreter.asm

```diff
@@ -703,6 +704,11 @@ end
 #         end
 #     )
 #
+# LLIntSlowPaths.h
+# extern "C" __attribute__((__used__)) __attribute__((visibility("hidden"))) void cProbeCallbackFunction(uint64_t i);
+# LLIntSlowPaths.cpp:
+# extern "C" void cProbeCallbackFunction(uint64_t i) {}
+#
```

Comment:
What's the intent of this comment?  The LLInt `probe` macro below is not full featured like the JIT probe.  This is meant to be used as a convenience debugging only.  Unlike with JIT probes, the protocol of the LLInt probe C callback can take any form of arguments you want.  It's up to the probe user.

Reply:
The purpose is to make it easy to add a probe without having to figure out why you broke the linker

Reply:
So, you're documenting an example here?  Please indent this by 4 spaces to match the probe example above.  I think the visual cue helps provide context.  Thanks.